documentation/source/contributing/contribute-art.rst

```diff
@@ -0,0 +1,4 @@
+Contribute art
```

Comment:
A bit missing some more details like what quality do we expect, which formats etc.

Reply:
All of this is documented on the website wiki. I will add a little text which hints this.